examples/website/src/shared/blocks/peerPageBreak.stories.tsx

```diff
@@ -10,15 +10,29 @@ storiesOf('Blocks|PeerPageBreak*', module)
   .add('Default', () => (
     <PageBreakBlock
       text={'Unterstuetze uns!'}
+      richText={[]}
```

Comment:
We don't update the stories in the examples website.

Reply:
so can i removed it completely?

Reply:
yes

Reply:
deleted